packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -162,7 +162,10 @@ const Button = ({
         fullWidth={isFullWidth}
         id={id}
         onClick={onClick}
-        ref={localButtonRef}
+        ref={(element: HTMLButtonElement) => {
+          muiProps?.ref(element);
```

Comment:
So I think I understand why we need this line. At the start of this function component, there's the line:
`const muiProps = useMuiProps();`, which creates a context for this component.
My assumption is, this context does not have the proper `ref` value at the time this component is going to be rendered out. Is that correct?

Reply:
From what I can tell, yes, that is correct